logging/Cargo.toml

```diff
@@ -14,7 +14,21 @@ doctest = false
 testutils = ["soroban-sdk/testutils"]
 
 [dependencies]
-soroban-sdk = { workspace = true }
+soroban-sdk = "0.7.0"
 
 [dev_dependencies]
-soroban-sdk = { workspace = true, features = ["testutils"] }
+soroban-sdk = { version = "0.7.0", features = ["testutils"] }
```

Comment:
I think it's worth investigating this.  It's a pain to have to change the deps for specific examples.

Reply:
Yeah, the workspace deps have to be used everywhere. If that didn't work for you before, chances are the workspace dependency was outdated.